### PR TITLE
[Feature/isxcsp 325]

### PR DESCRIPTION
**Pull request by leroj3**
Take a look at the picture below, that's what I implemented.
It will only show if the IKEA store is currently closed. For that, I created a dummy API with the different hours and days.

inlineMessage

### CODE REVIEW

File: src/pages/get-support/GetSupport.jsx [hunk 1]
```diff
@@ -26,12 +26,50 @@ const GetSupport = () => {
...
+  closedAt: {
+    mondayToFriday: 21,
+    saturday: 18,
+    sunday: 17
+  },
+  inlineMessage: {
+    headline: "Our customer center is now closed",
+    message: "Please come back at a later time and we will help you. In the meantime maybe you can find the answer to your question in our commonly asked questions."
+  }
+};
```
Comment (yawsoningka) [inline src/pages/get-support/GetSupport.jsx:51]: You could move this into the **services/mock** folder

File: src/pages/get-support/GetSupport.jsx [hunk 2]
```diff
@@ -26,12 +26,50 @@ const GetSupport = () => {
...
+console.log(afterHoursAPI.day[0], "afterHoursAPI.day[0]")
```
Comment (yawsoningka) [inline src/pages/get-support/GetSupport.jsx:52]: We can get rid of this console log

**Review by yawsoningka — CHANGES_REQUESTED** [review]

File: src/pages/get-support/GetSupport.jsx (hunk 2 above)
Comment (leroj3) [reply src/pages/get-support/GetSupport.jsx]: OK

File: src/pages/get-support/GetSupport.jsx (hunk 1 above)
Comment (leroj3) [reply src/pages/get-support/GetSupport.jsx]: OK

**Review by yawsoningka — APPROVED** [review]

**Pull request merged**